### PR TITLE
Add Postgres landing page, performance content, query reference

### PR DESCRIPTION
**Pull request by danieltprice**
Adds a Postgres landing page:
https://neon-next-git-dprice-add-postgres-landing-page-neondatabase.vercel.app/docs/postgres/postgres-intro

Adds a query performance guide:
https://neon-next-git-dprice-add-postgres-landing-page-neondatabase.vercel.app/docs/postgres/query-performance

Adds a query reference:
https://neon-next-git-dprice-add-postgres-landing-page-neondatabase.vercel.app/docs/postgres/query-reference

@YanicNeon @atumark Any queries you would like to see added to the page above?

Adds PgHero to the Monitoring page:
https://neon-next-git-dprice-add-postgres-landing-page-neondatabase.vercel.app/docs/introduction/monitoring#pghero

### CODE REVIEW

File: content/docs/postgres/query-performance.md [hunk 1]
```diff
@@ -0,0 +1,446 @@
...
+ ... 
+```
+
+For a description of each metric, refer to the official Postgres documentation: [The pg_stat_statements View](https://www.postgresql.org/docs/current/pgstatstatements.html#PGSTATSTATEMENTS-PG-STAT-STATEMENTS).
+
+<Admonition type="note" title="WHAT’S THE PERFORMANCE IMPACT OF PG_STAT_STATEMENTS?">
+Generally, `pg_stat_statements` is found to have a very small performance impact. Many users keep it installed so that it’s available when needed. For a discussion on this topic, please see this [Database Administrators Stack Exchange article](https://dba.stackexchange.com/questions/303503/what-is-the-performance-impact-of-pg-stat-statements).
+</Admonition>
+
+After allowing time for statistics collection, you can run queries like these to identify opportunities for query optimization:
```
Comment (bgrenon) [inline content/docs/postgres/query-performance.md:68]: How much time? Is it obvious that something is still running, or ?

File: content/docs/postgres/query-performance.md [hunk 2]
```diff
@@ -0,0 +1,446 @@
+---
+title: Optimize Postgres query performance
+subtitle: Learn about strategies for optimizing Postgres query performance
+enableTableOfContents: true
+---
+
+Many factors can impact query performance in Postgres, ranging from insufficient indexing and database maintenance to poorly optimized queries or inadequate system resources. With such a wide range of factors, it can be difficult to know where to start. In this topic, we'll look at several strategies you can use to optimize query performance in Postgres.
+
+- [Gather statistics](#gather-statistics)
```
Comment (bgrenon) [inline content/docs/postgres/query-performance.md:9]: Is there a loose or implied order in this list? Any opportunity to group into categories of some kind? Analysis or tuning, for example? 

Maybe a future improvement for this page would be to get some visual schematic of these operations, to help a reader see options at a glance.

**Review by bgrenon — APPROVED** [review]
Excellent stuff. There seems a minor issue with the TOC behavior with:
- title: Optimize queries
      slug: /postgres/query-performance
    - title: Query reference
      slug: /postgres/query-reference
    - title: Compatibility
      slug: reference/compatibility
The TOC is not expanding for these page or showing bolded when in focus.

File: content/docs/postgres/query-performance.md (hunk 1 above)
Comment (danieltprice) [reply content/docs/postgres/query-performance.md:68]: Change to "After allowing enough time for your workload to run fully and generate the necessary statistics"

File: content/docs/postgres/query-performance.md (hunk 2 above)
Comment (danieltprice) [reply content/docs/postgres/query-performance.md:19]: Categorized the strategies as suggested. We'll do a visual schematic with the next iteration. Good idea!

**Pull request merged**